site/layouts/_default/examples.html

```diff
@@ -31,6 +31,28 @@
           font-size: 3.5rem;
         }
       }
+
+      .b-example-divider {
```

Comment:
@ffoodd I didn't see it at first but the sidebars example displays vertical separators. So I've created here a common class grouping together color, box shadow and border. And then 2 separator classes allowing to have an horizontal one and a vertical one.

There are several ways of doing it: only use 2 classes, only put the `.b-example-divider` horizontal in this file and let examples like sidebars overwrite it to make a vertical one, etc. I'd like your thoughts.

Reply:
I think `.b-example-hr` could be dropped. Its `height` can be the default, and only `.b-example-vr` would override it.

Apart from this suggestion, it looks fine.

Reply:
Done in 1b51e6d8de6f194dac87849a7cac7f4c3908fbb0